e2e/cypress/integration/archived_channel/archive_channel_post_spec.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channel
+
+import {testWithConfig} from '../../../support/hooks';
```

Comment:
Correct path:
`import {testWithConfig} from '../../support/hooks';`

Reply:
Based on the folder structure this is actually right from this folder. I am not a fan of relative URLs when it exceeds more than `../` but as I mostly saw the usage with the relative path I went with it.

Reply:
@sudheerDev the Cypress test does not runs as the path is not correct. Can you please fix the correct path to `support/hooks`. It should be `../../support/hooks`. Only then the test works fine. Else we see this error. 

![Screenshot 2020-09-11 at 4 50 43 PM](https://user-images.githubusercontent.com/1429138/92919903-f566c180-f44e-11ea-930b-f9578ad86b1c.png)

Reply:
Wierd it was working for me earlier.